src/lib/components/permissions/row.svelte

```diff
@@ -38,6 +38,8 @@
             <div>Guests</div>
         {:else if role === 'any'}
             <div>Any</div>
+        {:else if !role.startsWith('user') && !role.startsWith('team') }
+            <div>Users/Verfified</div>
```

Comment:
> This should probably be `{role}` instead of `Users/Verfified` since we want to accept whatever the user sets as custom role

Yeah, makes sense. Made changes